bookwyrm_backend/src/main/java/com/bookwyrm/backend/book/controller/BookController.java

```diff
@@ -21,12 +21,21 @@
 
```

Comment:
Tests will also need to be written to ensure there is no regression on this change, probably by testing the sorting and searching differences.